<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicePortInformation</class>
 <widget class="QDialog" name="DevicePortInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device interface information </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device IP address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipAddress_lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Refresh interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interval_comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="reflesh_Btn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
